src/System.Collections.Immutable/ref/System.Collections.Immutable.cs

```diff
@@ -115,6 +115,10 @@ public partial struct ImmutableArray<T> : System.Collections.Generic.ICollection
         bool System.Collections.IList.IsFixedSize { get { throw null; } }
         bool System.Collections.IList.IsReadOnly { get { throw null; } }
         object System.Collections.IList.this[int index] { get { throw null; } set { } }
+#if FEATURE_ITEMREFAPI
```

Comment:
Either add a new define for these, or rename the existing one to be more generic.